.editorconfig

```diff
@@ -277,7 +277,7 @@ dotnet_diagnostic.SA1409.severity = none            # Remove unnecessary code.
 dotnet_diagnostic.SA1410.severity = none            # Remove delegate parenthesis when possible.
 dotnet_diagnostic.SA1411.severity = none            # Attribute constructor should not use unnecessary parenthesis.
 dotnet_diagnostic.SA1412.severity = none            # Store files as UTF-8 with byte order mark.
-dotnet_diagnostic.SA1413.severity = none            # Use trailing comma in multi-line initializers.
+dotnet_diagnostic.SA1413.severity = error            # Use trailing comma in multi-line initializers.
```

Comment:
This should rather be a warning (as every rule we use) to allow discoverability during development, but don't force you to comply with the rules when you're just prototyping. The CI build then treats warnings as errors, so they should still be adhered to before PR merge.

Reply:
I agree with Zoltan, this could be treated as a suggestion/warning

Reply:
Personally I hate it when the project build locally but fails the CI because the rules are different. I rather see it failing locally so we can fix it before the CI runs.  But if you want to change it to warning, feel free.

Reply:
Having errors like this enforces perfectly formatted code in every single scenario. It doesn't let you quickly try out things and don't care about formatting until you actually open a PR.